src/Katip/Scribes/Raven.hs

```diff
@@ -26,11 +28,13 @@ mkRavenScribe sentryService permitItem verbosity = return $
         name = sentryName $ Katip._itemNamespace item
         level = sentryLevel $ Katip._itemSeverity item
         msg = show $ Katip._itemMessage item
+        katipAttrs =
+          foldMap (\loc -> [("loc", Aeson.toJSON $ Katip.Core.LocJs loc)]) (Katip._itemLoc item)
         updateRecord record = record
             { Raven.srEnvironment = Just $ toS $ Katip.getEnvironment $ Katip._itemEnv item
-            , Raven.srTimestamp = show $ Katip._itemTime item
+            , Raven.srTimestamp = Katip._itemTime item
             -- add katip context as raven extras
-            , Raven.srExtra = extras $ Katip.payloadObject verbosity $ Katip._itemPayload item
+            , Raven.srExtra = HM.fromList $ map (first T.unpack) $ (katipAttrs ++) $ HM.toList $ Katip.payloadObject verbosity $ Katip._itemPayload item
```

Comment:
Arrow makes this code unnecessary complex

Reply:
`first` is Functor-level complexity and has very little to do with arrows. What do you propose?

Reply:
To leave the code as it were?

Reply:
The `Value -> ByteString` bit has to be removed in order to comply with the new `raven-haskell`, so that's not really an option.

Reply:
sure, let's leave that out